bedrock/firefox/templates/firefox/browsers/quantum.html

```diff
@@ -15,20 +15,19 @@
 
 {% block content %}
 
-{% call hero(
-  title=ftl('the-latest-firefox'),
-  class='mzp-has-image mzp-t-firefox mzp-t-product-firefox',
-  image_url='img/firefox/browsers/quantum/browser.jpg',
-  include_highres_image=True,
-  include_cta=True,
-  heading_level=1
-) %}
-<p>{{ ftl('firefox-quantum-was') }}</p>
-{{ download_firefox_thanks() }}
-<p><a class="mzp-c-cta-link" href="https://blog.mozilla.org/blog/2017/11/14/introducing-firefox-quantum/?utm_source=mozilla.org-firefox-browsers-quantum&utm_medium=referral&utm_campaign=seo-quantum&amp;utm_content=hero&amp;entrypoint=mozilla.org-firefox-browsers-quantum">{{ ftl('learn-more-about') }}</a></p>
-{% endcall %}
-
 <div class="mzp-l-content">
```

Comment:
unrelated to this PR, but I felt like the hero for the recent firefox/browsers/quantum page should be inside the `mzp-l-content` 🤷‍♀️

Reply:
The hero component already has an `mzp-l-content` container within it, so nesting it inside another one here creates some odd looking cropped content at smaller screen widths

Perhaps using one of the non-default [container theme classes](https://protocol.mozilla.org/patterns/templates/content-container.html), such as `mzp-t-content-xl` might look closer to how you want?

![image](https://user-images.githubusercontent.com/400117/97280341-108d6500-1834-11eb-9ec3-d344e177904b.png)